client/src/components/ResourceManager.jsx

```diff
@@ -2,14 +2,72 @@
 
 import React, { useEffect, useState } from 'react';
 import { Menu } from 'antd';
-import { DownOutlined, EditOutlined, CloseOutlined } from '@ant-design/icons';
+import {
+  DownOutlined,
+  UpOutlined,
+  EditOutlined,
+  CloseOutlined,
+  PlusOutlined,
+} from '@ant-design/icons';
 
 import { getCategories } from '../utils/api';
 
 import '../css/ResourceManager.css';
 
-const ResourceManager = () => {
+type Props = {
+  categories: { [string]: Array<string> },
+  categoryName: String,
+};
+
+const SidebarCategory = (props: Props) => {
   const { SubMenu } = Menu;
+
+  const { categories, categoryName } = props;
+  const [opened, setOpened] = useState(false);
+
+  const handleChange = () => {
+    setOpened(!opened);
+  };
+
+  return (
+    <SubMenu
+      {...props}
+      title={
+        <span>
+          {categoryName}
+          <span style={{ position: 'absolute', right: 15 }}>
```

Comment:
Ideally we remove all inline styling and put it in a separate css file, but that can be for another day.